microsoftgraph/microsoft-graph-explorer-v4
# Fix: Component name on device characteristics event

## Overview
Closes #2172 

### Demo
![image](https://user-images.githubusercontent.com/45680252/196394873-bd91762c-cd55-43f3-a67f-fa5ba607959c.png)


### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* How to test this PR
Click on the Run Query button
Open the network tab and open the track API call
Notice that the device characteristics payload has a component name
